Code/6-wire-version/lib/BWC/BWC_common.cpp

```diff
@@ -650,6 +650,7 @@ String BWC::getJSONStates() {
   doc["AIR"] = _cio.states[BUBBLESSTATE];
   doc["GRN"] = _cio.states[HEATGRNSTATE];
   doc["RED"] = _cio.states[HEATREDSTATE];
+  doc["HET"] = _cio.states[HEATSTATE];
```

Comment:
For consistency, can you change to "HTR" instead?

Reply:
Sure i've changed it